legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/tools/SessionProvider.java

```diff
@@ -134,7 +134,17 @@ public static Session getSessionUsingGitlabPersonalAccessToken(HttpServletReques
         GitlabPersonalAccessTokenProfileCreator creator = new GitlabPersonalAccessTokenProfileCreator(client.host);
         GitlabPersonalAccessTokenAuthenticator authenticator = new GitlabPersonalAccessTokenAuthenticator(client.scheme, client.host, client.apiVersion);
         GitlabPersonalAccessTokenCredentials credentials = extractor.extract(context);
-        authenticator.validate(credentials, context);
+
+        try
+        {
+            authenticator.validate(credentials, context);
+        }
+        catch (Exception e)
+        {
+            LOGGER.error("Validation failed for PA token: {}", e.getMessage());
```

Comment:
I think we should log the exception fully in case it's not the kind of exception we expect.
```suggestion
            LOGGER.error("Validation failed for PA token", e);
```